Buildings/Controls/OBC/ASHRAE/G36/AHUs/MultiZone/VAV/Economizers/Controller.mo

```diff
@@ -6,18 +6,19 @@ block Controller
     "Design of minimum outdoor air and economizer function";
   parameter Buildings.Controls.OBC.ASHRAE.G36.Types.BuildingPressureControlTypes buiPreCon
     "Type of building pressure control system";
-  parameter Buildings.Controls.OBC.ASHRAE.G36.Types.EnergyStandard eneStd=Buildings.Controls.OBC.ASHRAE.G36.Types.EnergyStandard.Not_Specified
+  parameter Buildings.Controls.OBC.ASHRAE.G36.Types.EnergyStandard eneStd=
+    Buildings.Controls.OBC.ASHRAE.G36.Types.EnergyStandard.ASHRAE90_1
     "Energy standard, ASHRAE 90.1 or Title 24";
   parameter Buildings.Controls.OBC.ASHRAE.G36.Types.ControlEconomizer ecoHigLimCon
     "Economizer high limit control device";
-  parameter Buildings.Controls.OBC.ASHRAE.G36.Types.ASHRAEClimateZone ashCliZon=
-    Buildings.Controls.OBC.ASHRAE.G36.Types.ASHRAEClimateZone.Not_Specified
+  parameter Buildings.Controls.OBC.ASHRAE.G36.Types.ASHRAEClimateZone ashCliZon(
+    start=Buildings.Controls.OBC.ASHRAE.G36.Types.ASHRAEClimateZone.Zone_3A)
```

Comment:
What is the rationale for providing a `start` value? 
This value is not meaningful for many climate zones, hence it would be good to understand why we don't leave it unassigned or `Not_Specified`.

Reply:
@mwetter If the parameter has no `start` attribute, is unassigned and disabled, the model will fail to run. For example the parameter `ashCliZon`, if the Title 24 standard is used, the parameter will be disabled. When it is unassigned and there is no `start` attribute, we can not run the model.

The `Not_Specified` should not be an option for the climate zone, as there is no warning in the `ctrl-flow` to avoid user to choose the option.